tests/SprykerEcoTest/Zed/Braintree/_support/BraintreeBusinessTester.php

```diff
@@ -29,7 +37,77 @@ class BraintreeBusinessTester extends Actor
 {
     use _generated\BraintreeBusinessTesterActions;
 
-   /**
-    * Define custom actions here
-    */
+    /**
+     * @param array $params
+     *
+     * @return \Braintree\Transaction
+     */
+    public function getSuccessfulTransaction($params = []): Transaction
+    {
+        $defaultParams = [
+           'id' => 123,
+           'processorResponseCode' => '1000',
+           'processorResponseText' => 'Approved',
+           'createdAt' => new DateTime(),
+           'status' => 'authorized',
+           'type' => 'sale',
+           'amount' => 0,
+           'merchantAccountId' => 'abc',
+           'statusHistory' => new StatusDetails([
+               'timestamp' => new DateTime(),
+               'status' => 'authorized',
+           ]),
+        ];
+
+        return Transaction::factory(array_merge($defaultParams, $params));
+    }
+
+    /**
+     * @param array $params
+     *
+     * @return \Braintree\Result\Successful
+     */
+    public function getSuccessfulTransactionResponse($params = []): Successful
+    {
+        return new Successful($this->getSuccessfulTransaction($params));
+    }
+
+    /**
+     * @param bool $isSuccess
+     *
+     * @return \Generated\Shared\Transfer\BraintreeTransactionResponseTransfer
+     */
+    public function getBraintreeTransactionResponseTransfer(bool $isSuccess): BraintreeTransactionResponseTransfer
```

Comment:
```suggestion
    public function createBraintreeTransactionResponseTransfer(bool $isSuccess): BraintreeTransactionResponseTransfer
```

but I would say this method is not needed at all

Reply:
We use it twice with different `$isSuccess` param value.